src/FS-FileDirectory-Adapters.package/FSFileDirectoryAdapter.class/instance/deleteDirectory..st

```diff
@@ -0,0 +1,3 @@
+as yet unclassified
```

Comment:
Please categorize appropriately.

Reply:
In my defense, `#createDirectory:` & Co. were not categorized either. :-) Anyway, fixed via 2dde141.

Reply:
Oops. Thank you!